src/Samples/NewWindow.Core/App.fs

```diff
@@ -21,34 +21,42 @@ type AppMsg =
   | Window2Close
   | Window2Msg of Window2Msg
 
-
 module App =
-  module Window1 =
-    let get m = m.Window1
-    let set v m = { m with Window1 = v }
-    let map = map get set
-  module Window2 =
-    let get m = m.Window2
-    let set v m = { m with Window2 = v }
-    let map = map get set
-    let mapOutMsg = function
+  module App_Window1 =
+    let get_Window1State (m: App) : WindowState<string> =
+      m.Window1
+    let set_Window1State (v: WindowState<string>) (m: App) : App =
+      { m with Window1 = v }
+    let map_Window1State : ((WindowState<string> -> WindowState<string>) -> App -> App) =
+      map get_Window1State set_Window1State
+  module App_Window2 =
+    let get_Window2 (m: App) : Window2 option =
+      m.Window2
+    let set_Window2 (v: Window2 option) (m: App) : App =
+      { m with Window2 = v }
+    let map_Window2 : ((Window2 option -> Window2 option) -> App -> App) =
```

Comment:
This function names are worse.  Please revert these.

Reply:
I don't think renaming the functions is a good idea. If the names of get/set/map functions aren't clear enough, we should replace them with lambdas (which I think is what many other F# users do). Adding parent module names to function names just repeats information; it doesn't add any.

We could add `RequireQualifiedAccess` to the get/set/map modules if that would help.

Reply:
I think adding `RequireQualifiedAccess` is a good compromise here.

Reply:
Yes, requiring qualified access is good. Qualified access was my intention.

Reply:
I reverted to the original `get`, `set`, and `map`.